app/src/main/res/values/strings.xml

```diff
@@ -704,24 +704,24 @@
         <item quantity="other">%d selected</item>
     </plurals>
     <string name="feed_group_dialog_empty_name">Empty group name</string>
-    <string name="feed_group_dialog_delete_message">Do you want to delete this group?</string>
+    <string name="feed_group_dialog_delete_message">Delete this group?</string>
     <string name="feed_create_new_group_button_title">New</string>
     <string name="feed_group_show_only_ungrouped_subscriptions">Show only ungrouped subscriptions</string>
     <string name="settings_category_feed_title">Feed</string>
     <string name="feed_update_threshold_title">Feed update threshold</string>
     <string name="feed_update_threshold_summary">Time after last update before a subscription is considered outdated — %s</string>
     <string name="feed_update_threshold_option_always_update">Always update</string>
-    <string name="feed_load_error">Error loading feed</string>
+    <string name="feed_load_error">Could not load feed</string>
     <string name="feed_load_error_account_info">Could not load feed for \'%s\'.</string>
-    <string name="feed_load_error_terminated">The author\'s account has been terminated.\nNewPipe will not be able to load this feed in the future.\nDo you want to unsubscribe from this channel?</string>
+    <string name="feed_load_error_terminated">The author\'s account has been terminated.\nNewPipe will not be able to load this feed in the future.\nUnsubscribe from this channel?</string>
     <string name="feed_load_error_fast_unknown">The fast feed mode does not provide more info on this.</string>
     <string name="feed_use_dedicated_fetch_method_title">Fetch from dedicated feed when available</string>
     <string name="feed_use_dedicated_fetch_method_summary">Available in some services, it is usually much faster but may return a limited amount of items and often incomplete information (e.g. no duration, item type, no live status)</string>
-    <string name="feed_use_dedicated_fetch_method_enable_button">Enable fast mode</string>
-    <string name="feed_use_dedicated_fetch_method_disable_button">Disable fast mode</string>
-    <string name="feed_use_dedicated_fetch_method_help_text">Do you think feed loading is too slow? If so, try enabling fast loading (you can change it in settings or by pressing the button below).\n\nNewPipe offers two feed loading strategies:\n• Fetching the whole subscription channel, which is slow but complete.\n• Using a dedicated service endpoint, which is fast but usually not complete.\n\nThe difference between the two is that the fast one usually lacks some information, like the item\'s duration or type (can\'t distinguish between live videos and normal ones) and it may return less items.\n\nYouTube is an example of a service that offers this fast method with its RSS feed.\n\nSo the choice boils down to what you prefer: speed or precise information.</string>
+    <string name="feed_use_dedicated_fetch_method_enable_button">Fast mode</string>
```

Comment:
```suggestion
    <string name="feed_use_dedicated_fetch_method_enable_button">Fast loading</string>
```

Reply:
This is an action, and thus needs a verb